src/cmd/services/m3coordinator/ingest/m3msg/ingest.go

```diff
@@ -203,7 +211,9 @@ func (op *ingestOp) ingest() {
 		op.p.Put(op)
 		return
 	}
+	age := xtime.Now().Sub(xtime.UnixNano(op.metricNanos))
```

Comment:
do we know the metricNanos is from the first original metric before pre-aggregation?

Reply:
Good question, let me test this out.